selenium_tests/course_info/page_objects/course_info_detail_page_object.py

```diff
@@ -51,19 +51,12 @@ def go_to_people_page(self):
     def reset_form(self):
         self.find_element(*Locators.RESET_FORM_BUTTON).click()
 
-    def is_locator_element_present(self, locator_element):
-        try:
-            self.find_element(locator_element)
-        except NoSuchElementException:
-            return False
-        return True
-
     def verify_buttons_to_edit_page_are_present(self):
-
-        if (self.is_locator_element_present(Locators.RESET_FORM_BUTTON) and
-            self.is_locator_element_present(Locators.SUBMIT_FORM_BUTTON)):
+        try:
+            (self.find_element(*Locators.RESET_FORM_BUTTON).is_displayed()
```

Comment:
Great, nice catch!  I thought the  try/except would replace the need for the "if" in this case (since if either element is not displayed it fails out "Except" and return False.  But it's probably much cleaner to add in the "if".  Fixed and thanks!